connector/connect/client/jvm-internal/pom.xml

```diff
@@ -0,0 +1,235 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this is mostly a copy from jvm/pom.xml, so can surely be pruned as jvm-internal should have less deps.